Jenkinsfile

```diff
@@ -12,6 +12,10 @@ elifePipeline {
                 checkout scm
                 sh "IMAGE_TAG=${commit} docker-compose build"
                 image = DockerImage.elifesciences(this, "pattern-library", commit)
+                elifePullRequestOnly { prNumber ->
+                    def assetsImage = DockerImage.elifesciences(this, "pattern-library_assets", commit)
+                    assetsImage.tag("pr-${prNumber}").push()
```

Comment:
Might be ok to do this before the tests so we can use the PR even if it's not yet green. Maybe add a comment to say this is intended.